src/app/server/CommissioningWindowManager.h

```diff
@@ -96,6 +96,8 @@ class CommissioningWindowManager : public SessionEstablishmentDelegate, public a
 
     CHIP_ERROR OpenCommissioningWindow();
 
+    CHIP_ERROR ListenForPASE();
```

Comment:
Could you document what this does? Like, how do you "stop listening for PASE"? How does it related to OpenCommissioningWindow, etc?

Reply:
Added, but then removed the whole function because someone else added one that was nearly identical.